migrations/800_groups__add.sql

```diff
@@ -0,0 +1,29 @@
+-- A Work in Progress to discuss shared schema for group storage in PL
+
+CREATE TABLE IF NOT EXISTS group_configs (
+    id BIGSERIAL PRIMARY KEY,
+    course_instance_id BIGINT NOT NULL REFERENCES course_instances(id) ON DELETE CASCADE ON UPDATE CASCADE,
+    assessment_id BIGINT REFERENCES assessments(id) ON DELETE CASCADE ON UPDATE CASCADE,
+    type TEXT, -- what will these be? This might belong in groups rather than groups_config
+    name TEXT, -- or maybe description is accurate here? Maybe as a base for groups created from it?
+    minimum INT,
+    maximum INT,
```

Comment:
Is there a use case where we'd want to impose these restrictions on the number of users in a group? I'm guessing `maximum` is more for students opting-in to a group, to prevent them from making a group that's too large.

Reply:
Yes, part of their design includes defined max and minimums for groups (regardless of if they're opted-in or auto assigned.)